api/v2/performanceprofile_types.go

```diff
@@ -73,8 +73,7 @@ type CPU struct {
 	// except the reserved CPUs. In order to guarantee that your workload will run on the isolated CPU:
```

Comment:
I misread this statement and though it was misleading. It actually isn't, the key here is "in order to guarantee". Then the suggestions are valid because a system configured like that indeed should guarantee that the workload runs on isolated CPUs. Removed my previous comment which added suggestions based on my wrong assumption.

Reply:
let's make sure all the cases are covered: https://github.com/openshift-kni/performance-addon-operators/pull/434